### PR TITLE
138 - PV/Radar - Superadmin adds generic Radar group

### PR DESCRIPTION
**Pull request by richardli5945**
Added "Renal Groups" functionality in PV, Radar can get new groups from this change.

### CODE REVIEW

File: patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupAddAction.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+public class GroupAddAction extends Action {
+
+    public ActionForward execute(
+        ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response)
+            throws Exception {
+
+        Unit checkExistUnit = LegacySpringUtils.getImportManager().retrieveUnit(
+                BeanUtils.getProperty(form, "unitcode"));
+        if (checkExistUnit != null) {
```
Comment (denizozger) [inline patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupAddAction.java:49]: Kevin: In the future can you leave some spaces where you find appropriate? ie. before and after "if" statements, before "return" statements, to seperate block of code with unrelated logic etc.

File: patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupsDisplayAction.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+public class GroupsDisplayAction extends Action {
+
+    public ActionForward execute(
+            ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response)
+            throws Exception {
+
+        User user = LegacySpringUtils.getUserManager().getLoggedInUser();
+
+        if (LegacySpringUtils.getUserManager().getCurrentSpecialtyRole(user).equals("superadmin")) {
+            List<Unit> items = LegacySpringUtils.getUnitManager().getAll(null, new String[]{"radargroup"});
```
Comment (denizozger) [inline patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupsDisplayAction.java:48]: Could "units" be a better name?

Comment (richardli5945) [inline patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupsDisplayAction.java:48]: yes, I have modified it and pushed it already.

File: patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupAddAction.java (hunk 1 above)
Comment (richardli5945) [inline patientview-parent/patientview/src/main/java/org/patientview/patientview/group/GroupAddAction.java:49]: ok, I will follow you in the future task.

**Pull request merged**